03_launch_mgmt_cluster.sh

```diff
@@ -195,14 +213,23 @@ function launch_cluster_api_provider_baremetal() {
       touch capbm.err.log
       if [ "${CAPI_VERSION}" == "v1alpha1" ]; then
         kubectl scale statefulset cluster-api-provider-baremetal-controller-manager -n metal3 --replicas=0
-      elif [ "${CAPI_VERSION}" == "v1alpha2" ]; then
+      elif [[ "${CAPI_VERSION}" == "v1alpha2" ]] || [[ "${CAPI_VERSION}" == "v1alpha3" ]]; then
         kubectl scale -n metal3 deployment.v1.apps capbm-controller-manager --replicas 0
       fi
       nohup make run >> capbm.out.log 2>> capbm.err.log &
     fi
     popd
 }
 
+function apply_cert_manager() {
+  if [ "${CAPI_VERSION}" == "v1alpha3" ]; then
+    kubectl create namespace cert-manager
+    kubectl apply --validate=false -f https://github.com/jetstack/cert-manager/releases/download/v0.13.0/cert-manager.yaml
+    sleep 60
+  fi
+}
+
```

Comment:
this is now part of the make deploy command for CAPBM, it does not need to be here

Reply:
removed